modules/ocl/cl_image_360_stitch.cpp

```diff
@@ -470,8 +470,12 @@ CLImage360Stitch::calc_fisheye_initial_info (SmartPtr<VideoBuffer> &output)
             _fisheye[i].handler->set_output_size (_fisheye[i].width, _fisheye[i].height);
         }
 
-        for(int i = 0; i < _fisheye_num; i++) {
-            _stitch_info.merge_width[i] = XCAM_ALIGN_UP((uint32_t)(20.0f / 360.0f * out_info.width), 32);
+        int idx_next;
+        for (int i = 0; i < _fisheye_num; i++) {
+            idx_next = (i == (_fisheye_num - 1)) ? 0 : (i + 1);
+
+            _stitch_info.merge_width[idx_next] = _fisheye[i].width / 2 + _fisheye[idx_next].width / 2 - out_info.width / 4;
```

Comment:
out_info.width / 4,   is 4 a hardcode for _fisheye_num? if so, please remove this hardcode.